dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```
@@ -42,7 +42,7 @@ disallowServeServletsByClassName.name=Disallow serve servlet by class name
 disallowServeServletsByClassName.desc=Disallows the use of serveServletsByClassnameEnabled on the application server level. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disallowserveservletsbyclassname.
 
 doNotServeByClassName.name=Do not serve by class name
-doNotServeByClassName.desc=A semi-colon delimited list of classes to be completely disallowed from being served by classname. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.donotservebyclassname.
+doNotServeByClassName.desc=A semi-colon delimited list of classes to be completely disallowed from being served by class name. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.donotservebyclassname.
```

this is a little unclear-what does "by class name" modify? Also, unless there is a functional difference between "separated" and "delimited", use one term consistently


```suggestion
doNotServeByClassName.desc=A semicolon-separated list of classes by class name that are prohibited from being served. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.donotservebyclassname.
```